composer.json

```diff
@@ -40,7 +40,10 @@
             "src/Helpers/Form/Template/Utils/Frontend.php",
             "src/Helpers/Form/Utils.php",
             "src/Helpers/Frontend/Shortcode.php",
-            "src/Helpers/Utils.php"
+            "src/Helpers/Utils.php",
+            "src/Helpers/Session/Session.php",
+            "src/Helpers/Session/Donation.php",
+            "src/Helpers/Session/DonationConfirmation/Frontend.php"
```

Comment:
We should really start to build out static facades sooner than later. We're loading _all_ of these files on every request and the `use function ...` at the top of the files is getting crazy.

Reply:
@JasonTheAdams Helper function which is added to this pr converted into there own class and facades. I will create separate issue to convert other helper functions.